content/articles/getting-started-with-nuxt-3/index.md

```diff
@@ -0,0 +1,154 @@
+
+### Table of contents
++ [Nuxt 3](#nuxt-3)
++ [Starting a new project](#starting-a-new-project)
++ [Nuxt 3 features](#nuxt-3-features)
++ [Nuxt 3 plugins](#nuxt-3-plugins)
++ [Why you need to use Nuxt 3](#why-you-need-to-use-nuxt-3)
+
+### Prerequisites
+To get started, knowledge of JavaScript concepts will be of much help. Also, make sure these tools are installed on your computer:
++ [Node.js](https://nodejs.org/en/download/)
++ [Visual Studio Code Editor](https://code.visualstudio.com/download)
++ [Volar extension](https://marketplace.visualstudio.com/items?itemName=johnsoncodehk.volar)
+
+### Nuxt 3
+Nuxt 3 is an open-source framework for web development that is both simple and powerful. Nuxt 3 was developed by Sébastien Chopin to emulate the functionality of [Nuxt.js](https://vueschool.io/lessons/what-is-nuxtjs), but with Vue instead of React. It is entirely written in typescript for easy maintenance and built-in typescript support.
+
+Nuxt 3 is currently in beta. It is not yet production-ready. Let's take a look at what's available.
+
+![Features comparison](/engineering-education/getting-started-with-nuxt-3/comparison.jpg)
+
+[Image source](https://v3.nuxtjs.org/getting-started/introduction/#comparison)
+
+There are two ways you can use to create your Nuxt 3 application. You can use Nuxt 3 or Nuxt Bridge.  Nuxt Bridge allows you to upgrade your existing Nuxt 2 application while waiting for Nuxt 3 to be ready for production. As you are aware of the distinctions between Nuxt 2, Nuxt Bridge, and Nuxt 3, it provides you with some fascinating features. You can see that Nuxt 3 is currently unstable, so obviously, you will use it when it's production-ready.
+
+If you want to use Nuxt Bridge, it's semi-stable, so it's better. You can see the performance is faster and a lot more features in Nuxt 3. You will learn about Nuxt 3 features later in this article. Go through this [migration guide](https://v3.nuxtjs.org/getting-started/migration/) to learn how to upgrade from Nuxt 2 to Nuxt 3.
+
+### Starting a new project
+Note that Nuxt 3 only supports Node Version 14 or 16.
+
+Open a VS Code terminal and type the following command to create a new project. 
+`npx nuxi init nuxt3`
+`nuxt3` is our project name.
+
+The next step is to install the dependencies in the generated `package.json` file. First, `cd` into the nuxt3 directory
+`cd nuxt3` then install the dependencies with `npm i` or `yarn i`. In our scenario, we'll use:
+`npm i`
+
+![Install Nuxt 3](/engineering-education/getting-started-with-nuxt-3/install-nuxt3.jpg)
+
+Once it finishes installing, we can go ahead and start our project. To start a local development server, run this command:
+`npm run dev`
+
+![Initialize the project](/engineering-education/getting-started-with-nuxt-3/start-project.jpg)
+
+If all works well, copy and open the [localhost] (http://localhost:3000/) link on your browser, and there is our Nuxt 3 project.
```

Comment:
@EKahuha Check for links. I think this one is broken.

Reply:
> @EKahuha Check for links. I think this one is broken.

The link will work if you've installed Nux 3 and followed the steps above.

Reply:
The formatting is what i mean. There shouldn't be any spaces in between []&()